meta-iot2000-example/recipes-node/node-red-contrib-modbus/node-red-contrib-modbus_5.13.3.bb

```diff
@@ -3,10 +3,12 @@ SUMMARY = "The all in one Modbus TCP and Serial contribution package for Node-RE
 # WARNING: the following LICENSE and LIC_FILES_CHKSUM values are best guesses - it is
 # your responsibility to verify that the values are complete and correct.
 LICENSE = "BSD-3-Clause"
-LIC_FILES_CHKSUM = "file://LICENSE;md5=38c55ac37ed977af78e113caffd06846"
+LIC_FILES_CHKSUM = "file://LICENSE;md5=4ce3d0350364f3efff545d6ba54f61eb"
```

Comment:
And here as well: New licenses?

Reply:
Some new year range is added to copyright line, no new license.